docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -198,6 +198,49 @@ It quietly succeeds. Now `\d children` shows that the foreign key constraint `ch
 
 Add the specified constraint to the table.
 
+#### [*alter_column_type*]
+
+Change the type of an existing column. Depending if data on disk is required to be changed, different semantics apply.
+
+Indexes and simple table constraints involving the column will be automatically converted to use the new column type by reparsing the originally supplied expression. 
+
+The optional `COLLATE` clause specifies a collation for the new column; if omitted, the collation is the default for the new column type. 
+
+The optional `USING` clause specifies how to compute the new column value from the old; if omitted, the default conversion is the same as an assignment cast from old data type to new. 
+
+A `USING` clause must be provided if there is no implicit or assignment cast from old to new type.
```

Comment:
Could this be considered plagiarism?  It's word for word the same as PG docs.

Reply:
From their license: https://www.postgresql.org/about/licence/

> Permission to use, copy, modify, and distribute **this software and its documentation** for any purpose, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and this paragraph and the following two paragraphs appear in all copies.

Reply:
So do those things need to appear as code comments in this documentation md file?  In which case might as well come up with new wording instead.

Reply:
This is not resolved